test/ContentValidationListenerTest.php

```diff
@@ -129,7 +174,7 @@ public function testReturnsApiProblemResponseIfCollectionRequestWithoutBodyIsInv
         ]));
         $listener = new ContentValidationListener([
             'Foo' => ['GET' => 'FooValidator'],
-        ], $services);
+        ], $services, ['Foo' => 'foo_id']);
```

Comment:
Cool - so, essentially the test was not correctly written previously. With the additional tests you wrote, this becomes clearer. Thanks!